src/content/docs/get-started/boilerplate-project.mdx

```diff
@@ -125,55 +214,55 @@ The Edge Delivery components that provide the content and layout for non-commerc
 
 ## Start exploring
 
-Run your storefront locally using the `npm start` command and use the table below to see how each **boilerplate page** has a corresponding **Commerce block**, which wraps the **drop-in** (or widget) that provides the block with its features and functions.
+Run your storefront locally using the `npm start` command and use the table below to see how each **boilerplate page** has a corresponding **Commerce block**, which wraps the **drop-in** component that provides the block with its features and functions.
 
-:::note[Live Search components]
-The Gear page is rendered by the [Live Search Product Listing Page
-(PLP)](https://experienceleague.adobe.com/en/docs/commerce/live-search/live-search-storefront/plp-styling)
-and the Search box (🔍) is rendered by the [Live Search
-Popover](https://experienceleague.adobe.com/en/docs/commerce/live-search/live-search-storefront/storefront-popover).
-To learn more about the Live Search service, visit [Introduction to Live
-Search](https://experienceleague.adobe.com/en/docs/commerce/live-search/overview#live-search-components).
-:::
+<Aside type="note" title="New in the June 2025 suite">
+The boilerplate now includes four new drop-in components:
 
-### Relationship between the Commerce blocks and drop-in components
+- [Product Discovery](/dropins/product-discovery/)
+- [Product Recommendations](/dropins/recommendations/)
+- [Wishlist](/dropins/wishlist/)
+- [Personalization](/dropins/personalization/)
 
-The first column links to the Commerce block code in the boilerplate repository. The second column links to the drop-in/widget code that powers the block.
+Additionally, legacy widgets have been replaced with drop-in components for better maintainability and consistency.
+</Aside>
 
+### Relationship between the Commerce blocks and drop-in components
```

Comment:
The whole table is not 100% correct anymore, as some Blocks use multiple drop-ins/Containers. I think it's safe to remove it altogether.